android/app/src/main/java/io/mosip/residentapp/MyInjiPackage.java

```diff
@@ -0,0 +1,30 @@
+package io.mosip.residentapp;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class MyInjiPackage implements ReactPackage {
```

Comment:
Do we really need to use name as `My`?
Can we rename it to VciClientPackage or InjiVciClientPackage?

Reply:
Noted & changed